okhttp/src/main/java/okhttp3/CipherSuite.java

```diff
@@ -39,326 +37,334 @@
    */
   private static final ConcurrentMap<String, CipherSuite> INSTANCES = new ConcurrentHashMap<>();
 
-  // Last updated 2014-11-11 using cipher suites from Android 21 and Java 8.
+  // Last updated 2016-07-03 using cipher suites from Android 24 and Java 9.
 
-  // public static final CipherSuite TLS_NULL_WITH_NULL_NULL = of("TLS_NULL_WITH_NULL_NULL", 0x0000, 5246, MAX_VALUE, MAX_VALUE);
```

Comment:
GitHub is failing miserably at this diff so I’ve manually marked what’s changed! (Not much.)